src/main/java/io/stargate/sgv2/jsonapi/service/embedding/configuration/EmbeddingProviderConfigStore.java

```diff
@@ -26,10 +26,20 @@ public static ServiceConfig custom(Optional<Class<?>> implementationClass) {
     }
   }
 
-  record RequestProperties(int maxRetries, int retryDelayInMillis, int timeoutInMillis) {
+  record RequestProperties(
+      int maxRetries,
+      int retryDelayInMillis,
+      int timeoutInMillis,
+      Optional<String> requestTypeQuery,
```

Comment:
Needed to map 2 recently added request-properties to pass to embedding providers.